config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'landing_page#index'
+
+  post '/health_card', to: 'health_card#create', as: :issue_vc, format: :fhir_json
```

Comment:
The post is a fhirBundle, which right now is what the QBP client search result will return. The health_card controller packages the health card and does some manipulation for display (date formatting, eventually CVX display lookup). I'm thinking of it more as a backend-for-frontend pattern rather than data API.

Reply:
I was expecting the flow to look something like this amazing sequence diagram:
```
Front End     Back End     IIS     Converter
     ------------>
                 |
                 ----------->
                            |
                 <-----------
                 |
                 --------------------->
                                      |
                 <---------------------
                 |
     <------------
```
So the front end sends a query to the backend, and the backend queries the IIS system, does the conversion (assuming a successful query), and then passes the processed result to the front end.

Reply:
I think for the patient search form, it makes sense to get back a v2 search response and convert it to a FHIR bundle for further use, but having a single form controller do the search and create the health card seems like a lot packaged in one call. We might ultimately inject other steps in there, like select which health card you want (COV 19 vs other infectious disease) in which case a separate search controller and health card controller would be helpful.

Reply:
I think it does make sense to have this be a two step process. `GET Index` for search and `POST create` for the actual healthcard issuing. I agree with @nkarip that it gives us the most flexibility for multiple use cases.